takwimu/templates/takwimu/_includes/profile/country_selector.html

```diff
@@ -1,18 +1,35 @@
 {% load staticfiles %}
 
 <div class="country-selector btn-group">
-  <button class="btn btn-outline-secondary btn-lg dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-    <img class="rounded-circle border-light mr-2" src="{% static 'img/flags/nigeria.svg' %}"
-        border="8" style="border-style:double; width: 50px; height: 50px;"/>
-    Nigeria
+
+  <button class="btn btn-outline-secondary btn-lg dropdown-toggle" type="button"
+          id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true"
+          aria-expanded="false">
+    <img class="rounded-circle border-light mr-2"
+         src={% with 'img/flags/'|add:current_page.title|lower|add:'.svg' as image_static %}
+           {% static image_static %}
+         {% endwith %}
+             border="8"
+         style="border-style:double; width: 50px; height: 50px;"/>
+    {{ current_page.title }}
   </button>
-  <div class="dropdown-menu pt-0 rounded-0" aria-labelledby="dropdownMenuButton" style="max-width: 180px;">
+  <div class="dropdown-menu pt-0 rounded-0" aria-labelledby="dropdownMenuButton"
+       style="max-width: 180px;">
+
+    {% for profile in current_page.get_siblings %}
+      <a class="dropdown-item" href="{{ profile.get_url }}"
+         data-value="{{ profile.title | lower }}"><img
+          class="rounded-circle border-light mr-2"
+          src={% with 'img/flags/'|add:profile.title|lower|add:'.svg' as image_static %}
+            {% static image_static %}
+          {% endwith %} border="8"
+          style="border-style:double; width: 50px; height: 50px;"/>{{ profile.title }}
+      </a>
```

Comment:
Same here. Aren't quotes needed i.e `src="..."`?

Reply:
@DavidLemayian the static tag adds them automatically quotes, so adding them is redundant

Reply:
I stand corrected. All the examples show quotes but I suppose this is a force of habit.